app/services/atmosphere/appliance_vms_manager.rb

```diff
@@ -108,6 +108,14 @@ def appliance_satisfied(vm)
       appliance.state = :satisfied
       updater.update(new_vm: vm)
       @tags_manager.create_tags_for_vm(vm)
+      register_in_cep_engine(vm)
+    end
+
+    def register_in_cep_engine(vm)
```

Comment:
Unused method argument - `vm`. If it's necessary, use `_` or `_vm` as an argument name to indicate that it won't be used. You can also write as `register_in_cep_engine(*)` if you want the method to accept any arguments but don't care about them.